src/sentry/event_manager.py

```diff
@@ -1321,6 +1333,9 @@ def _handle_regression(self, group, event, release):
         elif GroupResolution.has_resolution(group, release):
             return
 
+        elif has_pending_commit_resolution(group):
```

Comment:
@getsentry/infrastructure fyi this is a change in event processing, though it should be minor impact

Reply:
also note this path still needs test coverage